addons/hr_timesheet/models/project_update.py

```diff
@@ -27,7 +27,7 @@ def _compute_display_timesheet_stats(self):
     def create(self, vals_list):
         updates = super().create(vals_list)
         encode_uom = self.env.company.timesheet_encode_uom_id
-        ratio = self.env.ref("uom.product_uom_hour").ratio / encode_uom.ratio
+        ratio = self.env.ref("uom.product_uom_hour").factor / encode_uom.factor
```

Comment:
wait why is this inconsistent with your change in `product_product`? i.e. why was one converted from * => /, but not the opposite?